### PR TITLE
[fix](doc) updated node version as per given in the url

### PR DESCRIPTION
**Pull request by Nitin-Kashyap**
## Proposed changes

a) updated the node version in text as per the URL in hyperlink for the same.
b) spell error fixed for toolchain

### CODE REVIEW

File: docs/zh-CN/docs/install/source-install/compilation-arm.md [hunk 1]
```diff
@@ -585,11 +585,11 @@ USE_AVX2=OFF sh build.sh
 
    - 问题原因
 
-     未使用 Ldb-Toolschain 进行编译
+     未使用 ldb-toolchain 进行编译
 
    - 解决方案
 
-     - 检查 Ldb-Toolschain 环境变量是否配置
+     - 检查 tdb-toolchain 环境变量是否配置
```
Comment (rohitrs1983) [inline docs/zh-CN/docs/install/source-install/compilation-arm.md:592]: please change to ldb-toolchain

Comment (Nitin-Kashyap) [reply docs/zh-CN/docs/install/source-install/compilation-arm.md]: fixed

**Review by rohitrs1983 — APPROVED** [review]

**Comment by luzhijing** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by httpshirley — APPROVED** [review]

**Pull request merged**